### ISSUE TITLE
Broken capture button for modifiers and more

### ISSUE THREAD

**Issue by aybe** [joncampbell123/dosbox-x#1954]
**Describe the bug**

It's impossible to say, remap shutdown to <kbd>Right Ctrl</kbd> + <kbd>F9</kbd>.

Problem is, capture button only captures <kbd>Right Ctrl</kbd> then stops, it doesn't recognizes it as a modifier.

So you can capture either <kbd>Right Ctrl</kbd> or <kbd>F9</kbd> but not both.

Also, the capture button is quite confusing in comparison to add button, we never know when it will actually work.
 
**Screenshots**

Default screen:
![dosbox-x_2020-10-16_21-58-50](https://user-images.githubusercontent.com/1537591/96303500-c964d000-0ffa-11eb-814d-b948f1b92542.png)

Here I press <kbd>F9</kbd> then have to press <kbd>ESC</kbd> 3 times, then nothing works, then mapper exits:
![2020-10-16_21-59-37](https://user-images.githubusercontent.com/1537591/96303566-e7323500-0ffa-11eb-8734-8aa931a280a7.gif)
(note that it's exactly the same whether or not you clicked `Add` before)

Only `Add` will capture and show <kbd>Right Ctrl</kbd>:
![2020-10-16_22-04-02](https://user-images.githubusercontent.com/1537591/96303910-8525ff80-0ffb-11eb-992d-41c0474a65eb.gif)

In short, capture is confusing and doesn't handle modifiers.

**Comment by Wengier** [comment]
@aybe I think to map a function to a shortcut like Ctrl + F9, you have to first select a modifier (Mod1 usually corresponds to Ctrl), and then add the key (F9 in this case). So modifier + key will becomes Ctrl + F9 for this function. But I think the mapper interface can indeed be improved somehow for better usability.

**Comment by aybe** [comment]
Yes I know `Mod1` is about that but the idea was to try map <kbd>Right Ctrl + F9</kbd>.

Maybe the command should detect it's a modifier and not stop capture ?

And the capture process is confusing, when is `Capt` supposed to work ? For `Add` it's perfectly clear when it does.

**Comment by Wengier** [comment]
@aybe In my opinion there should be a help screen for the mapper, as new users may simply have no idea how to start with the interface if there is no guide. To be honest, I don't know when when is ```Capt``` supposed to work either as I have never used it myself before indeed.

**Comment by aybe** [comment]
Yes, that's a good idea, not even sure if the button works at all, it just does nothing.
